charts/playbooks-ai/templates/diagnose-kubernetes.yaml

```diff
@@ -0,0 +1,55 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flanksource/duty/main/schema/openapi/playbook.schema.json
+{{- if and .Values.diagnose.enabled }}
+---
+apiVersion: mission-control.flanksource.com/v1
+kind: Playbook
+metadata:
+  name: diagnose-unhealthy-kubernetes-resource
```

Comment:
Why does this need to be restricted to Kubernetes resources ?   Or even unhealthy resources?